zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -1054,30 +1055,28 @@ impl<F: Field> SubCircuit<F> for BytecodeCircuit<F> {
 
     /// Return the minimum number of rows required to prove the block
     fn min_num_rows_block(block: &witness::Block) -> (usize, usize) {
-        if block.bytecode_map.is_some() {
-            // when enable feature "dual_bytecode", get two sets of bytecodes here.
-            let (first_bytecodes, second_bytecodes) = block.get_bytecodes_for_dual_sub_circuits();
-            let minimum_row: usize = max(
-                first_bytecodes
-                    .iter()
-                    .map(|bytecode| bytecode.bytes.len() + 1)
+        #[cfg(feature = "dual_bytecode")]
```

Comment:
one of `bytecode_map` functionality is used to check if configure feature `dual_bytecode`. code like:
if bytecode_map.is_some(){
 // enable dual_bytecode` feature case
}else{
// default : no feature case
}
this can save multiple code `cfg(feature)` statement like used here.
#[cfg(feature = "dual_bytecode")] { 
....
}
#[cfg(not(feature = "dual_bytecode"))]{
.....
}